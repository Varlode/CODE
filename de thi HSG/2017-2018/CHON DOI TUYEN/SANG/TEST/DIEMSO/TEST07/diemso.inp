1000
745035 374759 56366 786107 796885 36228 167220 738470 153222 146903 746714 302159 960197 112857 337300 959238 732806 780461 803840 727527 125566 245320 551419 867755 344545 92808 148500 963820 811120 655545 991371 517971 473631 159475 633005 422388 108313 836163 959195 391712 519061 427433 67713 580049 959856 915761 216958 750122 109003 3419 166178 50702 196381 441790 596100 213667 39445 149487 229185 282991 755011 256003 373380 159985 41452 968380 652927 692026 754024 739162 733611 530006 219302 437435 878955 882741 653698 457536 603571 380300 590799 338938 902449 821484 292927 174003 413813 926517 54273 948389 593286 19 984026 222177 320307 556893 867485 893809 203333 430357 580528 707804 346723 874104 551600 962478 192431 445006 806247 147126 233862 878603 742410 707871 379248 630116 925724 699793 366711 295415 59423 153653 997452 303043 68438 401615 800325 990270 740863 194218 630244 576381 249868 906387 701422 494523 354021 583054 810288 819136 703492 499414 512992 424779 630930 967624 474467 451641 68332 45812 245404 951099 64750 616184 593374 272610 956578 767464 55823 646980 340168 849116 478778 628321 769799 197452 57497 469202 929274 684549 410963 14782 425655 279360 551654 687809 936359 295135 437226 581072 271658 579528 898003 499262 58082 630234 617508 259183 227507 681339 878435 757632 581551 869823 206468 278239 614621 244969 908229 693390 106046 290274 552673 299504 29066 964827 752701 922127 768205 138617 916010 691151 263643 57155 981781 870322 195143 152763 873698 851659 555686 560593 151021 613383 199860 154699 183603 476748 741973 49106 379338 778769 752777 897733 554833 709265 142089 745324 985169 530483 918277 561081 982113 605078 334563 369748 242472 721160 48233 354342 733813 331497 410599 724364 275346 1138 597163 956274 249494 793149 354641 468846 523122 29418 979385 469516 984658 308534 809350 404093 297712 515956 11610 87929 853951 756624 528167 265198 257691 667979 735031 855044 652996 273357 284176 42035 250460 262917 711801 660056 394220 629098 553485 512209 392955 276656 30071 117078 633475 453162 326646 646056 315615 565699 526530 298900 371787 698735 88922 104266 535551 772743 272939 300575 677784 582161 536000 133146 471867 930188 429063 767198 143834 920893 956778 218109 6866 657387 176386 829929 200689 896980 870082 384930 357153 940057 266018 129135 795355 402113 374789 490260 424493 511015 325523 157385 972917 806708 445290 806858 819800 71074 847118 591030 97234 616965 991956 340886 272673 586032 749996 146298 605221 947848 223288 731569 412749 182450 106582 198591 127702 94204 432911 854138 869160 685181 954304 143489 118348 517384 213016 489590 677782 859768 954015 355579 203526 657331 159935 63977 755346 244769 393752 805175 426555 507372 924130 139698 661624 765360 557799 440142 386957 491551 783514 81936 635203 709837 193424 269266 334981 207521 661182 43311 42340 582574 759309 83970 501173 994040 91255 469492 452038 795966 572166 65537 951086 990992 726770 819500 228920 762949 804963 423957 624935 215978 662901 847452 406863 599175 783235 601677 143214 581264 487060 751617 284691 591149 715816 59729 779266 680878 479697 438748 900760 642115 405785 811975 818346 938162 979376 992201 641278 439789 119398 464447 717523 897143 542467 355620 598485 680617 812215 597136 552667 760751 962136 848901 963317 115165 869848 303177 607226 706974 168245 985049 976601 457542 917533 407178 433348 877451 714925 272348 350796 76270 151819 885196 782210 177666 999096 417024 286635 505452 913188 908344 257340 727328 800405 497034 830937 101588 218958 6951 387495 996043 232374 613916 569530 237993 92130 753619 305795 448180 119890 782873 888218 141592 283681 670551 100994 339739 77663 757436 712671 903281 210372 911588 989998 186298 914025 35479 16345 245909 727770 594800 259266 876035 922825 54987 822155 613968 785604 269526 780386 201785 269294 497235 629354 651572 9072 248719 398678 166976 591021 848588 750369 268148 228324 123754 177278 407628 663402 43819 4445 866329 776513 230437 16728 473917 975877 24256 940606 91679 554587 325209 320172 840948 348981 274544 135914 873788 637673 726191 368154 847278 79462 656102 586156 130815 673017 535971 568514 570560 609764 223050 819641 318802 697439 912266 520679 774389 675716 204245 590979 80233 71840 991863 964629 106367 858645 699044 358251 72270 2410 288044 60446 183376 19263 966618 153310 475698 392360 902228 403751 978695 969288 824089 406853 11681 474705 928912 843537 591547 678228 639774 800634 379827 492443 63080 168100 943461 797330 455565 237171 329627 14952 108726 930808 372344 628796 533055 837835 265848 53336 514854 69038 730089 523572 170835 852160 891933 983382 700600 157441 207888 813447 152212 458859 703832 648270 212033 245465 973159 739673 685307 931834 224390 333969 198720 146444 539776 597124 20432 484664 917896 94273 37355 542449 341705 895787 413425 652933 818138 652151 457405 379239 830470 780795 158882 756026 31856 575865 297572 138737 305830 592427 807954 703282 856478 300876 622628 638070 318157 774241 305136 959738 632936 920717 692263 675492 564698 560054 138042 771493 733351 980280 872276 134710 533249 728859 390531 265062 530716 53379 705441 976567 180574 922951 643701 817863 827944 836730 162448 291654 968744 384375 533332 949906 225345 527006 770510 784310 163574 174293 904780 669633 646520 809711 547776 302186 215302 735927 16992 419130 755785 715620 316565 367435 158110 414661 241846 223578 249363 140720 537388 806234 364969 449717 154941 936360 135243 600872 799591 188496 898870 587989 33843 672340 63507 810945 124399 466530 551772 531499 742761 521528 848223 459075 322544 817168 160880 158817 720610 94522 596894 697852 25308 614200 743263 417572 616323 428333 201266 91550 799461 624353 469758 983352 100079 631284 245020 746985 109100 733390 336681 693964 197946 789453 258298 102901 466644 669267 196501 601233 234405 723796 543711 218347 417800 388937 332444 417595 128936 641041 427418 413039 599933 551109 629619 651955 438740 217461 606200 349101 106040 929944 298997 803351 710015 522610 720762 571080 137208 445101 793511 130624 126240 590630 901204 756140 77533 413027 862997 690996 190284 658137 186840 517962 589575 578961 776267 549037 555995 61180 875393 654818 542246 119235 518626 222118 81787 518664 874200 333967 476441 202925 147768 959208 594785 2274 743610 508491 189891 522608 783008 827066 122332 746660 387996 333693 15489 332249 947639 423096 794976 154650 340801 401108 826797 206089 490434 896130 435610 822919 441125 144934 940066 270438 53952 756787 393022 195175 619019 451133 155095 712355 778813 70323 588596 91098 798453 280473 850685 964436 910646 300422 582086 347183 879979 615733 521916 682264 475524 559029 661351 537083 61202 981563 94158 293166 872679 126184 458741 494847 649423 256478 805150 487141 214137 354486 415790 107097 274639 244010 872394 650075 40882 155004 121292 514559 672271 819948 598463 166240 288429 518657 480677 57805 757731 
3
58 108
97 125
49 187
