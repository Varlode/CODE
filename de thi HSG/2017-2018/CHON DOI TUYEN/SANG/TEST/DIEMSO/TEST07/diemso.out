27027126
15450793
70019756
